### PR TITLE
User Guide changes

### PR DESCRIPTION
**Pull request by intelmark**
Added several line breaks to the user gude
Touched up layer hierarchical diagrams to improve alignment and correct
layer numbers

Signed-off-by: Mark Feldman <mark.feldman@intel.com>

### CODE REVIEW

**Comment by tianjunwork** [comment]
Docs/3-layer-hierarchical-low-delay2.png the first I frame is layer 0

**Comment by tianjunwork** [comment]
what's changed in Docs/3-layer-hierarchical-low-delay3.png ?

File: Docs/svt-hevc_encoder_user_guide.md [hunk 1]
```diff
@@ -278,9 +278,9 @@ The encoder parameters present in the Sample.cfg file are listed in this table b
...
 | **HierarchicalLevels** | -hierarchical-levels | [0 – 3] | 3 | 0 : Flat<br>1: 2-Level Hierarchy<br>2: 3-Level Hierarchy<br>3: 4-Level HierarchyMinigop Size = (2^HierarchicalLevels) <br>(e.g. 3 == > 7B pyramid, 2 ==> 3B Pyramid)Refer to Appendix A.1 |
 | **BaseLayerSwitchMode** | -base-layer-switch-mode | [0,1] | 0 | 0 : Use B-frames in the base layer pointing to the same past picture<br>1 : Use P-frames in the base layerRefer to Appendix A.1 |
-| **PredStructure** | -pred-struct | [0 – 2] | 2 | 0: Low Delay P1: Low Delay B<br>2: Random AccessRefer to Appendix A.1 |
-| **IntraPeriod** | -intra-period | [-2 - 255] | -2 | Distance Between Intra Frame inserted. -1 denotes no intra update. -2 denotes auto. |
-| **IntraRefreshType** | -irefresh-type | [1,2] | 1 | 1: CRA (Open GOP)2: IDR (Closed GOP) |
+| **PredStructure** | -pred-struct | [0 – 2] | 2 | 0: Low Delay P<br>1: Low Delay B<br>2: Random AccessRefer to Appendix A.1 |
+| **IntraPeriod** | -intra-period | [-2 - 255] | -2 | Distance Between Intra Frame inserted. <br>-1 denotes no intra update. <br>-2 denotes auto. |
```
Comment (tianjunwork) [inline Docs/svt-hevc_encoder_user_guide.md]: Pls add <br> before "Refer to"

File: Docs/svt-hevc_encoder_user_guide.md [hunk 2]
```diff
@@ -278,9 +278,9 @@ The encoder parameters present in the Sample.cfg file are listed in this table b
 | **SeparateFields** | -separate-fields | [0,1] | 0 | 1 : Interlaced input, application will separate top and bottom fields and encode it as progressive. <br>0 : Treat video as progressive video |
 | **HierarchicalLevels** | -hierarchical-levels | [0 – 3] | 3 | 0 : Flat<br>1: 2-Level Hierarchy<br>2: 3-Level Hierarchy<br>3: 4-Level HierarchyMinigop Size = (2^HierarchicalLevels) <br>(e.g. 3 == > 7B pyramid, 2 ==> 3B Pyramid)Refer to Appendix A.1 |
 | **BaseLayerSwitchMode** | -base-layer-switch-mode | [0,1] | 0 | 0 : Use B-frames in the base layer pointing to the same past picture<br>1 : Use P-frames in the base layerRefer to Appendix A.1 |
```
Comment (tianjunwork) [inline Docs/svt-hevc_encoder_user_guide.md]: Pls add <br> before "Refer to"

Comment (tianjunwork) [inline Docs/svt-hevc_encoder_user_guide.md]: Pls add <br> before "Refer to"

File: Docs/svt-hevc_encoder_user_guide.md [hunk 3]
```diff
@@ -278,9 +278,9 @@ The encoder parameters present in the Sample.cfg file are listed in this table b
 | **SeparateFields** | -separate-fields | [0,1] | 0 | 1 : Interlaced input, application will separate top and bottom fields and encode it as progressive. <br>0 : Treat video as progressive video |
 | **HierarchicalLevels** | -hierarchical-levels | [0 – 3] | 3 | 0 : Flat<br>1: 2-Level Hierarchy<br>2: 3-Level Hierarchy<br>3: 4-Level HierarchyMinigop Size = (2^HierarchicalLevels) <br>(e.g. 3 == > 7B pyramid, 2 ==> 3B Pyramid)Refer to Appendix A.1 |
```
Comment (tianjunwork) [inline Docs/svt-hevc_encoder_user_guide.md]: Pls add <br> before "Refer to" and "Minigop"

File: Docs/svt-hevc_encoder_user_guide.md (hunk 1 above)
Comment (tianjunwork) [reply Docs/svt-hevc_encoder_user_guide.md]: while it doesn't show the text. it is a new line. br.

Comment (intelmark) [reply Docs/svt-hevc_encoder_user_guide.md]: what's changed in Docs/3-layer-hierarchical-low-delay3.png ? - some baseline adjustments for the text so they align better

**Pull request merged**